### DISCUSSION TITLE
Hidden pages

### DISCUSSION THREAD

**Discussion by couleurm** [squidfunk/mkdocs-material#6616] [Q&A] [answered]
Hey, I'd like to make pages accessible at a certain url that are not on the `nav:`, is that possible even with a little hack?

**Comment by alexvoss** [comment, marked as answer]
You can leave them out of the `nav`. If you want to get rid of the warning messages that MkDocs would normally generate, you need to [declare them explicitly as not included in the nav](https://www.mkdocs.org/user-guide/configuration/#not_in_nav).
